huaweicloud/services/elb/resource_huaweicloud_elb_pool.go

```diff
@@ -118,6 +99,49 @@ func ResourcePoolV3() *schema.Resource {
 					},
 				},
 			},
+			"name": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"type": {
```

Comment:
it's better to put this parameter together with loadbalancer_id and listener_id.

Reply:
done